polaris-react/src/components/Box/Box.scss

```
@@ -0,0 +1,114 @@
+.root {
```

Is this a new pattern we want to adopt? Typically the root class is named after the component

```suggestion
.Box {
```